scripts/enable-datastore-viewer.sh

```diff
@@ -53,32 +54,32 @@ if [ ! -e /etc/nginx/sites-enabled ]; then
         exit 1
 fi
 
-# Get the list of running applications, and corresponding ports.
-ps ax | grep appserver | grep -Ev '(grep|appscaledashboard)' | grep -- '--admin_port' | sed 's;.*--admin_port \([0-9]*\).*/var/apps/\(.*\)/app .*;\1 \2;g' | sort -ru | while read port app_id; do
-        # Enable only for specified apps.
-        if [ -n "$APPS_ID" ]; then
-                found="no"
-                for x in $APPS_ID ; do
-                        if [ "$x" = "$app_id" ]; then
-                                found="yes"
-                                break
-                        fi
-                done
-                if [ "$found" = "no" ]; then
+APPENGINE_IP=""
+for x in $(cat /etc/appscale/all_ips); do
+        OUTPUT=$(ssh $x -i /etc/appscale/keys/cloud1/*.key 'ps ax | grep appserver | grep -Ev '(grep|appscaledashboard)' | grep -- "--admin_port" | sed "s;.*--admin_port \([0-9]*\).*/var/apps/\(.*\)/app .*;\1 \2;g" | sort -ru' | grep "$APP_ID")
+        for i in $OUTPUT ; do
+                if [ "$i" = "$APP_ID" ]; then
                         continue
+                else
+                        port=$i
+                        APPENGINE_IP=$x
+                        break
                 fi
-        fi
+        done
+done
 
-	let $((VIEWER_PORT += 1))
+let $((VIEWER_PORT += 1))
```

Comment:
This is not necessary if you start the VIEWER_PORT from 8100 at https://github.com/AppScale/appscale/pull/2171/files#diff-d438d935c16df0ab964b80d79ce47507L10